src/pages/taesooKim/Main/Feed.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import CommentList from "./CommentList";
+import BtnLike from "./BtnLike";
+import Like from "./Like";
+
+export class Feed extends Component {
+  constructor() {
+    super();
+    this.state = {
+      value: "",
+      comment: [],
+      isClicked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ comment: this.props.commentList });
+  }
+
+  getValue = e => {
+    this.setState({
+      value: e.target.value,
+    });
+  };
+
+  handleAdd = e => {
+    if (this.state.value) {
+      e.preventDefault();
+      const newComment = {
+        id: Date.now(),
+        userName: "orosy.ts",
+        content: this.state.value,
+        isUser: true,
+      };
+      const comment = [...this.state.comment, newComment];
+      this.setState({ comment, value: "" });
+    }
+    e.preventDefault();
+  };
+
+  handleDelete = id => {
+    const filteredComments = this.state.comment.filter(
+      comment => comment.id !== id
+    );
+    this.setState({ comment: filteredComments });
+  };
+
+  handleClick = () => {
+    this.setState({ isClicked: !this.state.isClicked });
+  };
+
+  render() {
+    const { value, isClicked, comment } = this.state;
+    const { name, profile, description, image, like, update } = this.props;
+    return (
```

Comment:
```suggestion
    const { name, profile, description, image, like, update } = this.props;
    
    return (
```
한줄 띄워주면 구분 더 쉬울거 같습니다

Reply:
네네 수정하였습니다🐸